Resources/Prototypes/Catalog/Fills/Books/bookshelf.yml

```
@@ -130,3 +130,94 @@
       - id: BookInspiration
         prob: 0.1
         orGroup: BookAuthor
+        # Corvax lore books
```

```suggestion
        # Corvax-Lore-Start
```
и оконичить `# Corvax-Lore-End`